lib/system/io.nim

```diff
@@ -931,14 +931,13 @@ iterator lines*(f: File): string {.tags: [ReadIOEffect].} =
   ## Iterate over any line in the file `f`.
   ##
   ## The trailing newline character(s) are removed from the iterated lines.
-  ## Example:
   ##
-  ## .. code-block:: nim
-  ##   proc countZeros(filename: File): tuple[lines, zeros: int] =
-  ##     for line in filename.lines:
-  ##       for letter in line:
-  ##         if letter == '0':
-  ##           result.zeros += 1
-  ##       result.lines += 1
+  runnableExamples:
+     proc countZeros(filename: File): tuple[lines, zeros: int] =
+       for line in filename.lines:
+         for letter in line:
+           if letter == '0':
+             result.zeros += 1
+         result.lines += 1
```

Comment:
```suggestion
  runnableExamples:
    proc countZeros(filename: File): tuple[lines, zeros: int] =
      for line in filename.lines:
        for letter in line:
          if letter == '0':
            result.zeros += 1
        result.lines += 1
```
When I say "ditto for the other example", I'm not just saying that for fun :P

Reply:
Oh I missed that.